src/actions/types.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  LOG_IN: 'LOG_IN',
+};
```

Comment:
@marcdomain @codeshifu, I'll want all types in one file. We will be adapting Marcus's